mixer/cmd/bundles.go

```diff
@@ -40,35 +41,28 @@ var addBundlesCmd = &cobra.Command{
 	Use:   "add [bundle(s)]",
 	Short: "Add clr-bundles to your mix",
 	Long:  `Add clr-bundles to your mix`,
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
@cmarcelo - just wanted to point out that this is a good example of a case where we do still want to use `RunE`. Cobra doesn't have a declarative way of saying "This command needs exactly one argument, except when a specific flag is passed", so that kind of logic has to live in the `Run` function. Using `RunE` here lets us pass that error back up to Cobra, and thus get usage data as expected, while still handling internal errors with `fail`.

Reply:
OK!